### PR TITLE
Fix some style nits in my previous commit

### PR DESCRIPTION
**Pull request by bernd-edlinger**

### CODE REVIEW

File: apps/s_socket.c [hunk 1]
```diff
@@ -187,17 +187,20 @@ int do_server(int *accept_sock, const char *host, const char *port,
...
         && BIO_ADDRINFO_socktype(next) == sock_type
         && BIO_ADDRINFO_protocol(next) == sock_protocol) {
-        if (sock_family == AF_INET && BIO_ADDRINFO_family(next) == AF_INET6) {
+        if (sock_family == AF_INET
+            && BIO_ADDRINFO_family(next) == AF_INET6) {
             sock_family = AF_INET6;
             sock_address = BIO_ADDRINFO_address(next);
         }
-        else if (sock_family == AF_INET6 && BIO_ADDRINFO_family(next) == AF_INET)
+        else if (sock_family == AF_INET6
```
Comment (levitte) [inline apps/s_socket.c]: While you're fixing nits, you might as well have this `else` join the `}` above

Comment (bernd-edlinger) [reply apps/s_socket.c]: sure, you mean " } else if ..." ?

Comment (levitte) [reply apps/s_socket.c]: Yes, exactly

File: apps/s_socket.c [hunk 2]
```diff
@@ -187,17 +187,19 @@ int do_server(int *accept_sock, const char *host, const char *port,
...
     next = BIO_ADDRINFO_next(res);
-    if(sock_family == AF_INET6)
+    if (sock_family == AF_INET6)
         sock_options |= BIO_SOCK_V6_ONLY;
     if (next != NULL
         && BIO_ADDRINFO_socktype(next) == sock_type
         && BIO_ADDRINFO_protocol(next) == sock_protocol) {
-        if (sock_family == AF_INET && BIO_ADDRINFO_family(next) == AF_INET6) {
+        if (sock_family == AF_INET
+            && BIO_ADDRINFO_family(next) == AF_INET6) {
```
Comment (richsalz) [inline apps/s_socket.c]: Since this is a style PR, please indent this line an extra tabstop

Comment (bernd-edlinger) [reply apps/s_socket.c]: okay, I assume also line 193..194 as well ?

Comment (bernd-edlinger) [reply apps/s_socket.c]: done, I will assume the approval is still valid.

**Comment by bernd-edlinger** [comment]
Merged.  Thanks!

**Pull request closed without merging**